app/src/main/java/com/accessibilitytraining/android/flow/detail/DetailAdapter.kt

```diff
@@ -34,8 +35,13 @@ class DetailAdapter : RecyclerView.Adapter<DetailAdapter.ImageViewHolder>() {
     class ImageViewHolder(private val itemBinding: ViewHolderImageBinding) :
         RecyclerView.ViewHolder(itemBinding.root) {
         fun onBindView(imageData: ListDataResponse.ListImageModel) {
-            itemBinding.tvTitle.text = imageData.title
-            itemBinding.tvDate.text = imageData.date
+            with(imageData){
+                itemBinding.apply {
+                    tvTitle.text = title
+                    tvDate.text = date
+                    AccessibilityBuilder().setText(String.format("%s, %s", title, date)).build(vRow)
```

Comment:
Should not use contentDescription to group the layout text
Better using the following code on "vRow"
"isFocusable = true" 
or
"View.importantForAccessibility = IMPORTANT_FOR_ACCESSIBILITY_YES" 

you can set them in xml also